# goal definition

string frame
string obj_name
std_msgs/Header header
geometry_msgs/Pose pick_pose
geometry_msgs/Pose place_pose

---

# result definition

---

# feedback
